scripts/bert/finetune_classifier.py

```diff
@@ -115,6 +115,9 @@
     type=int,
     default=128,
     help='Maximum length of the sentence pairs')
+parser.add_argument(
+    '--round_to', type=int, default=1,
+    help='The length of padded sequences will be rounded up to be multiple of this argument.')
```

Comment:
could you set default to None? 


The concept is new to many users. Could you add: when round to is set to 8, training throughput may increase for mixed precision training on GPUs with tensorcores.